alacritty/src/config/ui_config.rs

```diff
@@ -255,6 +255,7 @@ impl Default for Hints {
             enabled: vec![Hint {
                 content,
                 action,
+                persist: true,
```

Comment:
The default should be false. Most people don't want to open multiple URIs and this causes additional friction when having to dismiss the hints.